client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -91,6 +96,21 @@ const BrewRenderer = (props)=>{
 			...prevState,
 			viewablePageNumber : Math.floor(target.scrollTop / target.scrollHeight * rawPages.length)
 		}));
+
+		getCurrentPage(e);
+	};
+
+	const getCurrentPage = (e) => {
```

Comment:
Note, `viewablePageNumber` should already contain the current page in view, and it's already calculated during `handleScroll()`.

Feel free to rename `viewablePageNumber` to `currentPageNumber` if that makes more sense. But we don't need to calculate it twice.

Reply:
got rid of handleScroll() function entirely in favor of getCurrentPage().  handleScroll was giving me an issue, and getCurrentPage seems to function fine.